### PR TITLE
Add meetup and discord buttons

### PR DESCRIPTION
**Pull request by daynamariejpembie**
I created the buttons as shown below, The first button is when hovered and the second button is the original button design. Any feedback would be appreciated.
This is for issue #46 

image

This is the updated change:
image

### CODE REVIEW

**Comment by zarela** [comment]
@daynamariejpembie my feedback from the original PR dissapeared. 
When looking on the web, the buttons are too white and not completely visible, so we could do one of two things
- Change the buttons to be the reverse, so blue buttons and the hovers what you currently have or
- Keep everything as it is, but the buttons have a blue border.

File: package-lock.json
Comment (zarela) [inline package-lock.json:1]: Do we need this? Seems unrelated to youcr changes.

**Comment by jonathans199** [comment]
> @daynamariejpembie my feedback from the original PR dissapeared. When looking on the web, the buttons are too white and not completely visible, so we could do one of two things
> 
> * Change the buttons to be the reverse, so blue buttons and the hovers what you currently have or
> * Keep everything as it is, but the buttons have a blue border.

Or maybe for the white button add more width to the Border making it more visible and complete the task, we could over iterate on this and make it better.

**Comment by zarela** [comment]
@daynamariejpembie I still the changes to the package.json

**Review by zarela — APPROVED** [review]

**Comment by zarela** [comment]
closes #46
